test/e2e/ovn-kubernetes.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+    "folders": [
+        {
+            "path": "../../go-controller"
+        }
+    ]
+}
```

Comment:
what's this about ?

Reply:
Ah thats from my vscode, setup, will remove